normandy/bundler/models.py

```diff
@@ -14,3 +16,14 @@ def ids(self):
     def hash(self):
         recipe_hashes = '_'.join([recipe.content_hash for recipe in self.recipes])
         return hashlib.sha1(recipe_hashes.encode('utf-8')).hexdigest()
+
+    def __iter__(self):
+        for recipe in self.recipes:
+            yield recipe
+
+    @classmethod
+    def for_client(cls, client):
```

Comment:
A minor quibble: I think it would be better to put constructor-a-like methods like this right after `__init__`, order-wise.